### DISCUSSION TITLE
Reader Improvements

### DISCUSSION THREAD

**Discussion by lucdoebereiner** [Quill-OS/quill#12] [Ideas]
1.8. on Glo
- Scrolling an epub is quite hard in terms of the interface. Instead of moving the page, the dictionary keeps popping up. It takes me on average around 7 attempts.
- It would be great to be able to bookmark or remember the page last when one opened a book the last time. 
- Connected to the previous item, navigation within a book is hard. Scrolling through the book (the vertical scroll bar with page numbers) doesn't react
- It would be great to be able to navigated the toc

**Comment by tux-linux** [comment]
Hello,
No. 1: ePUBs aren't meant to be scrolled. You have to use the next and previous buttons. If sometimes text is missing on the screen, you can enable the "Show scrollbar as necessary" option in Settings.
If you mean that it's difficult to scroll there, you can click on the part of the scrollbar that isn't black, and it will probably be easier to move the bar.
No. 2: Disable "Global reading settings" in Settings
No. 3: This is not possible due to the fact that it would take ages to get the contents of one page. Imagine you're scrolling continuously, it would take several minutes for the reader to respond.
No. 4: That could be nice, I don't know how I'll get that to match the page number, though. I'll investigate.

I know I need to document certain things above, but sometimes you can look at the releases changelog, for example here with 1.8; https://github.com/Kobo-InkBox/inkbox/releases/tag/1.8 to get an idea of what are the new features.
"Global reading settings" was introduced in 1.7. Maybe could it be made a default option, but as far as I saw, for unknown reasons, the feature is unstable sometimes.
